pkg/crc/validation/validation.go

```diff
@@ -42,6 +42,14 @@ func ValidateDiskSize(value int) error {
 	return nil
 }
 
+func ValidatePersistentDiskSize(value int) error {
```

Comment:
maybe change this to also `validatePersistentVolumeSize` we are mixing `volume` and `disk` in the new names.. `persistentVolume` vs `persistentDisk`

Reply:
updated/